commit 488e73f28868c99c5b4b40d99dd4b3621e04bc93
Author: Ethan Zonca <e@ethanzonca.com>
Date:   Fri Feb 24 11:27:36 2017 -0500

    can: gs_usb: Don't use stack memory for USB transfers
    
    commit c919a3069c775c1c876bec55e00b2305d5125caa upstream.
    
    Fixes: 05ca5270005c can: gs_usb: add ethtool set_phys_id callback to locate physical device
    
    The gs_usb driver is performing USB transfers using buffers allocated on
    the stack. This causes the driver to not function with vmapped stacks.
    Instead, allocate memory for the transfer buffers.
    
    Signed-off-by: Ethan Zonca <e@ethanzonca.com>
    Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1304_488e73f28868c99c5b4b40d99dd4b3621e04bc93.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1304_488e73f28868c99c5b4b40d99dd4b3621e04bc93.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
